globalwarmingpotentials.csv

```diff
@@ -6,94 +6,97 @@
+Halon1202,,,,280,216,775,61.5,48.4,39.3
+CCl4,1400,1400,1730,2019,2200,3810,658,1990,810
+CH3Br,,5,2,3,2.43,8.74,0.692,0.535,0.438
+CH3CCl3,100,146,160,193,161,567,46,37.5,29.7
+HCFC21,,,148,179,160,575,29,35.5,45.6
+HCFC22,1500,1810,1760,2106,1960,5690,560,785,379
+HCFC123,90,77,79,96,90.4,325,25.8,20,16.4
+HCFC124,470,609,527,635,597,2070,170,143,110
+HCFC141b,600,725,782,938,860,2710,246,269,162
+HCFC142b,1800,2310,1980,2345,2300,5510,658,1430,514
+HCFC225ca,,122,127,155,137,491,39,30.4,24.8
+HCFC225cb,,595,525,633,568,1960,162,136,105
+HFC23,11700,14800,12400,13856,14600,12400,10500,15400,15100
+HFC32,650,675,677,817,771,2690,220,181,142
+HFC41,150,,116,141,135,485,38.6,30.4,24.6
+HFC125,2800,3500,3170,3691,3740,6740,1110,3300,1300
+HFC134,1000,,1120,1337,1260,3900,361,420,239
+HFC134a,1300,1430,1300,1549,1530,4140,436,733,306
+HFC143,300,,328,397,364,1300,104,82.6,66.6
+HFC143a,3800,4470,4800,5508,5810,7840,1940,5910,3250
+HFC152,,,16,20,21.5,77.6,6.14,4.74,3.89
+HFC152a,140,124,138,167,164,591,46.8,36.5,29.8
+HFC161,,,4,4,4.84,17.4,1.38,1.06,0.872
+HFC227ea,2900,3220,3350,3860,3600,5850,1100,3400,1490
+HFC236cb,,,1210,1438,1350,3750,387,620,268
+HFC236ea,,,1330,1596,1500,4420,428,572,288
+HFC236fa,6300,9810,8060,8998,8690,7450,6040,9200,8870
+HFC245ca,560,,716,863,787,2680,225,196,146
+HFC245fa,,1030,858,1032,962,3170,274,262,180
+HFC365mfc,,794,804,966,914,2920,261,272,172
+HFC4310mee,1300,1640,1650,1952,1600,3960,458,943,347
+SO2F2,,,4090,4732,4630,7510,1410,4360,1920
+SF6,23900,22800,23500,26087,25200,18300,34100,26200,30600
+NF3,,17200,16100,17885,17400,13400,18200,18200,20000
+CF4,6500,7390,6630,7349,7380,5300,10600,7660,9050
+C2F6,9200,12200,11100,12340,12400,8940,17500,12900,15200
+C3F8,7000,8830,8900,9878,9290,6770,12400,9660,11200
+cC4F8,8700,10300,9540,10592,10200,7400,22.3,17.3,14.1
+C4F10,7000,8860,9200,10213,10000,7300,13400,10400,12100
+C5F12,7500,9160,8550,9484,9220,6680,12700,9580,11200
+C6F14,7400,9300,7910,8780,8620,6260,11600,8960,10500
+C7F16,,,7820,8681,8410,6120,11300,8740,10200
+C8F18,,,7620,8456,8260,6010,11100,8590,10000
+C10F18,,,7190,7977,7480,5480,9780,7790,9010
+SF5CF3,,17700,17400,19396,18500,13900,21100,19300,21600
+cC3F6,,,9200,10208,,,,,
+HFE125,,14900,12400,13951,14300,13500,7680,15200,13100
+HFE134,,6320,5560,6512,6630,12700,1940,5530,2060
+HFE143a,,756,523,632,616,2170,173,143,113
+HCFE235da2,,350,491,595,539,1930,154,122,98.4
+HFE245cb2,,708,654,790,747,2630,213,174,137
+HFE245fa2,,659,812,981,878,3060,251,207,162
+HFE347mcc3,,575,530,641,576,2020,164,134,106
+HFE347pcf2,,580,889,1072,980,3370,279,237,181
+HFE356pcc3,,110,413,500,277,995,79,62.1,50.4
+HFE449s1,,297,421,509,460,1620,131,106,84.6
+HFE569sf2,,59,57,69,60.7,219,17.3,13.4,11
+HFE4310pccc124,,1870,2820,3353,3220,8720,920,1560,647
+HFE236ca12,,2800,5350,6260,6060,11700,1770,5020,1860
+HFE338pcc13,,1500,2910,3466,3320,9180,948,1520,657
+HFE227ea,,,6450,7377,7520,9800,2570,7720,4440
+HCFE236ea2,,,1790,2143,2590,7020,741,1260,521
```

Comment:
Was this a typo previously? I couldn't find HFE236ea2 in AR6...

Reply:
Seems likely! @jkikstra can you confirm/check with your CSV/Excel?

Reply:
Might be a typo indeed.
But not sure which is better to use.

HCFE-236ea2 is in AR6, both in the PDF and in the csv from @chrisroadmap.

Google tells me that we are probably dealing with Desfluran/Desflurane/Desfluoran here, double-checked with its formula CHF2OCHFCF3.

In a couple of european regulatory documents i found this as HFE-236ea2 ([umweltsbundeamt](https://www.umweltbundesamt.de/sites/default/files/medien/2503/dokumente/treibhauspotentiale_ausgewaehlter_verbindungen_und_deren_gemische.pdf), [Proposal for a Regulation of the European Parliament and of the Council on fluorinated greenhouse gases ](https://www.parlament.gv.at/PAKT/EU/XXVII/EU/02/42/EU_24287/imfname_10987241.pdf), [European Environment Agency, Annex II of Regulation (EU) No 517/2014 (not covered by old Regulation (EC) No 842/2006)](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjn9I246t_yAhW2hf0HHaIsC4kQFnoECAQQAQ&url=https%3A%2F%2Fwww.eea.europa.eu%2Fpublications%2Ffluorinated-greenhouse-gases-2020%2Fdownload&usg=AOvVaw3IpI8UZnGsfmsUdVuROZi2)), all quoting a GWP of 989, apparently based on AR4.

However, in a 2016 textbook called [Advances In Atmospheric Chemistry, Volume 1](https://books.google.at/books?id=3wGyDgAAQBAJ&pg=PA333&lpg=PA333&dq=HCFE-236ea2&source=bl&ots=n5XG6al9Gp&sig=ACfU3U0CsWrVRSXOlqePSiYOvZU__qyz_A&hl=en&sa=X&ved=2ahUKEwjn9I246t_yAhW2hf0HHaIsC4kQ6AF6BAgNEAM#v=onepage&q=HCFE-236ea2&f=false) it is called HCFE-236ea2 and the GWP (have not figured out what gwp yet, but probably safe to assume gwp100 now) is 1790.
Looks like that is the same as AR5.

Reply:
Thanks for digging into that!

If it's not used in SCMs or IAMs i'd consider removing it ...